README.md

```diff
@@ -48,22 +93,39 @@ if let foo = foo {
 }
 ```
 
-Alternatively, you might want to use Swift's Optional Chaining in some of these cases, such as:
+When dealing with `weak` or optional delegates, optional binding syntax is always preferable unless functionality is dependent on that value being unwrapped.
```

Comment:
This distinction is lost on me. one syntax is more compact than the other...maybe clarify some more?